build.sbt

```diff
@@ -4,8 +4,20 @@ lazy val scala213               = "2.13.7"
 lazy val scala212               = "2.12.15"
 lazy val supportedScalaVersions = List(scala213, scala212)
 
-organization := "com.sky"
-name         := "kafka-topic-loader"
+name                   := "kafka-topic-loader"
+organization           := "uk.sky"
+sonatypeCredentialHost := "s01.oss.sonatype.org"
+sonatypeRepository     := "https://s01.oss.sonatype.org/service/local"
+homepage               := Some(url("https://github.com/sky-uk/kafka-topic-loader"))
```

Comment:
lets put `"https://github.com/sky-uk/kafka-topic-loader"` in a `val` and reuse

Reply:
[`a410eb7` (#74)](https://github.com/sky-uk/kafka-topic-loader/pull/74/commits/a410eb7a94e454123e2aa6988695ca0415a570a7)